pl_bolts/models/detection/components/torchvision_backbones.py

```diff
@@ -0,0 +1,106 @@
+
+import torch.nn as nn
+from typing import Optional
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+try:
+    import torchvision
+
+    from ._supported_models import TORCHVISION_MODEL_ZOO
+except ModuleNotFoundError:
+    warn_missing_pkg('torchvision')  # pragma: no-cover
+
+
+__all__ = ["create_torchvision_backbone"]
+
+
+def _create_backbone_generic(model, out_channels: int):
```

Comment:
```suggestion
def _create_backbone_generic(model: nn.Module, out_channels: int):
```
or what is the type?